add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
```

Comment:
The current implementation lacks validation for the POST data. It's important to validate and sanitize user inputs to ensure data integrity and prevent potential security issues.

```php
$title = filter_input(INPUT_POST, 'title', FILTER_SANITIZE_STRING);
$author = filter_input(INPUT_POST, 'author', FILTER_SANITIZE_STRING);
$year = filter_input(INPUT_POST, 'year', FILTER_VALIDATE_INT);

if (!$title || !$author || !$year) {
    echo "Invalid input data.";
    exit;
}
```